src/lib/components/cloudprogressbar/CloudProgressBar.component.js

```diff
@@ -0,0 +1,115 @@
+//@flow
+import React from 'react';
+import styled, { css, keyframes } from "styled-components";
+import * as defaultTheme from "../../style/theme";
+
+export type CloudProgressBarProps = {
+  percentage: number,
+  borderSize: string,
+  cloudColor?: string,
+  progressCloudColor?: string,
+  children: Node
+};
+
+const keyFrameCloud = props => {
+  return keyframes`
+    from {
+      height:100%
+    } 
+    to {
+      height:${100 - props.percentage}%
+    }`;
+};
+
+const Container = styled.div`
+  ${props => {
+    return css`
+      width: 100%;
+      text-align: center;
+      position: relative;
+    `;
+  }}
+`;
+
+const ContainerProgress = styled.div`
+  ${props => {
+    return css`
+      animation-duration: 1s;
+      animation-fill-mode: both;
+      animation-name: ${keyFrameCloud};
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      width: 100%;
+      overflow: hidden;
+    `;
+  }}
+`;
+
+const Cloud = ({
+  strokeColor,
+  borderSize
+}) =>
+  <svg 
+    xmlns="http://www.w3.org/2000/svg" 
+    viewBox="0 2.8 16 10"
+    preserveAspectRatio="none"
+  >
+    <path 
+      d="m391.84 540.91c-.421-.329-.949-.524-1.523-.524-1.351 0-2.451 1.084-2.485 2.435-1.395.526-2.388 1.88-2.388 3.466 0 1.874 1.385 3.423 3.182 3.667v.034h12.73v-.006c1.775-.104 3.182-1.584 3.182-3.395 0-1.747-1.309-3.186-2.994-3.379.007-.106.011-.214.011-.322 0-2.707-2.271-4.901-5.072-4.901-2.073 0-3.856 1.202-4.643 2.925" 
+      transform="matrix(.77976 0 0 .78395-299.99-418.63)" 
+      fill="transparent"
+      stroke={strokeColor}
+      stroke-width={borderSize}
+      stroke-linejoin='round'
+    />
+  </svg>
+
+const CloudProgress = ({
+  strokeColor,
+  percentage,
+  borderSize
+}) =>
+  <ContainerProgress
+    percentage={percentage}
+  >
+    <svg 
+      xmlns="http://www.w3.org/2000/svg" 
+      viewBox="0 2.8 16 10"
+      preserveAspectRatio="none"
+    >
+      <path 
+        d="m391.84 540.91c-.421-.329-.949-.524-1.523-.524-1.351 0-2.451 1.084-2.485 2.435-1.395.526-2.388 1.88-2.388 3.466 0 1.874 1.385 3.423 3.182 3.667v.034h12.73v-.006c1.775-.104 3.182-1.584 3.182-3.395 0-1.747-1.309-3.186-2.994-3.379.007-.106.011-.214.011-.322 0-2.707-2.271-4.901-5.072-4.901-2.073 0-3.856 1.202-4.643 2.925" 
+        transform="matrix(.77976 0 0 .78395-299.99-418.63)" 
+        fill="transparent"
+        stroke={strokeColor}
+        stroke-width={borderSize}
+        stroke-linejoin='round'
+      />
+    </svg>
+  </ContainerProgress>
+
+const CloudProgressBar = ({
+  percentage = 0,
+  borderSize = '2px',
+  cloudColor = defaultTheme.brand.primary,
+  progressCloudColor = defaultTheme.brand.backgroundContrast2,
+  children = ''
```

Comment:
```suggestion
  children
```
IMO, it's not necessary to give a default value for children as it's a mandatory prop. 

Apart from that, it's better to specify the children in the storybook. The purpose of `storybook` is to give example/use case to the people who use this library.